pandas/_libs/index.pyx

```diff
@@ -1110,3 +1137,123 @@ cdef class ExtensionEngine(SharedEngine):
 
     cdef _check_type(self, object val):
         hash(val)
+
+
+cdef class MaskedIndexEngine(IndexEngine):
+
+    def __init__(self, ndarray values, ndarray mask):
+        super().__init__(values, mask)
```

Comment:
can you just remove `__init__` here?

Reply:
Forgot this, was necessary earlier. Removed